.circleci/config.yml

```diff
@@ -299,6 +299,22 @@ jobs:
             yarn storybook:ui:chromatic
       - store_test_results:
           path: test-results
+  chromatic-storybook-blocks:
+    executor:
+      class: medium+
+      name: sb_node_16_browsers
+    steps:
+      - git-shallow-clone/checkout_advanced:
+          clone_options: '--depth 1 --verbose'
+      - attach_workspace:
+          at: .
+      - run:
+          name: Running Chromatic
+          command: |
+            cd code
+            yarn storybook:blocks:chrmoatic
+      - store_test_results:
+          path: test-results
```

Comment:
This is the exact same steps as the `chromatic-storybook-ui` job above. we could squash them together as one job since they are so related, but we'd loose the concurrency.
Although I doubt these jobs are the bottlenecks anyway, so maybe it's okay to not do them concurrently?